src/main/java/org/generationcp/middleware/api/nametype/GermplasmNameTypeService.java

```diff
@@ -1,11 +1,24 @@
 package org.generationcp.middleware.api.nametype;
 
+import org.springframework.data.domain.Pageable;
+
 import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 
 public interface GermplasmNameTypeService {
```

Comment:
seems like some functions were moved from GermplasmServiceImpl but not deleted there so we have duplicated code? If that's the case, then remove them from GermplasmService and ask the QA to perform the regression tests needed.